addons/controllers/antrea/testcases/antrea-test-1.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: test-cluster-1
+  namespace: default
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks: [ "192.168.0.0/16" ]
+    services:
+      cidrBlocks: [ "192.168.0.0/16","fd00:100:96::/48" ]
+  infrastructureRef:
+    kind: ""
```

Comment:
@vijaykatam , I want to test `infrastructureRef.kind` field. For this, I am trying to give the following yaml 
```
apiVersion: cluster.x-k8s.io/v1beta1
kind: Cluster
metadata:
  name: test-cluster-1
  namespace: default
spec:
  clusterNetwork:
    pods:
      cidrBlocks: [ "192.168.0.0/16" ]
    services:
      cidrBlocks: [ "192.168.0.0/16" ]
  infrastructureRef:
    apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
    kind: DockerCluster
    name: test-cluster-1
---
apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
kind: DockerCluster
metadata:
  name: test-cluster-1
 ```
 But I am unable to find crd's which I can pass as external dependencies in GetExternalCRDPaths

Reply:
@vijaykatam Thanks for suggesting to add CRDs. I was not finding them initially. @blc1996  pointed me to those and I was able to run the use case.Thank you @blc1996 
I have updated the test case and now able to verify infraProvider as well along with other derived fields(serviceCIDR, serviceCIDRv6)
All the tests have been updated to check these fields.